frontend/resourceadm/components/ResourceDeployStatus/ResourceDeployStatus.tsx

```diff
@@ -64,7 +64,7 @@ export const ResourceDeployStatus = ({
       );
     }
     return error.map((e, index) => {
-      const textArr = e.message.split('"');
+      const textArr = e.message.split('\'');
```

Comment:
Should we try to deconstruct the array to make the code more readable? Not sure if the "text" variable is the best name. 🙈 
```suggestion
      const [linkButtonText, text]= e.message.split('\'');
```